src/locutus_util/ontology_api_etl.py

```diff
@@ -64,6 +86,53 @@ def collect_ols_data():
 
     return extracted_data
 
+
+def collect_umls_data():
+    """
+    Collects ontology data from the UMLS API.
+
+    Returns:
+        list: Transformed data from UMLS API.
+    """
+    logger.info(f"Fetching umls data")
+
+    data = fetch_data(umls_ontologies_url)
+    logger.info("Transforming umls data")
+
+    extracted_data = []  # Initialize a list to store the extracted data
+
+    while data:
+        results = data.get("result", [])
+        if not isinstance(results, list):
+            raise TypeError("Expected 'result' to be a list.")
+
+        for item in results:
+            # Transform the data into a consistent structure
+            extracted_data.append(
+                {
+                    "api_url": UMLS_API_BASE_URL,
+                    "api_id": "umls",
+                    "api_name": "UMLS Terminology Services",
+                    "ontology_code": item.get("abbreviation", ""),
+                    "curie": item.get("abbreviation", ""),
+                    "ontology_title": item.get("expandedForm", ""),
+                    "system": None,  # Not available at this time.
+                    "version": None,  # Not available at this time.
```

Comment:
I didn't see these in the UMLS response. I could backfill some of the systems from OLS most likely, or I'll have to get them from somewhere else. I'll ask Yelena if she requires the system